import.py

```diff
@@ -41,17 +41,19 @@ def main():
     if f[-4:] == ".shp":
       stem = f[:-4].replace(".", "_").replace("-","_")
       print("Opening shapefile:", stem)
-      reprojected = reprojectShapefile(f, dataDir, reprojectedDir, SRIDNamespace+":"+desiredSRID)
+      sf = shapefile.Reader(os.path.join(dataDir, f))
+      keyField = askUserForFieldNames(sf, stem)
+
+      reprojected = reprojectShapefile(f, stem, dataDir, reprojectedDir, SRIDNamespace+":"+desiredSRID, keyField)
       simplified = simplifyShapefile(reprojected, simplifiedDir, simplificationTolerance)
       sf = shapefile.Reader(simplified)
```

Comment:
Yes - it's just a throwaway pointer, and the processed shapefile ultimately stands in for the raw one.